mods/ts/weapons/healweapons.yaml

```diff
@@ -9,6 +9,9 @@
 		DebugOverlayColor: 00FF00
 		Damage: -5000
 		ValidTargets: Heal
+	Warhead@2Eff: FlashTargetsInRadius
+		ValidTargets: Infantry
+		ValidRelationships: Ally
```

Comment:
shouldn't the heal warhead then only heal allies as well?

Reply:
armaments have TargetRelationships: Ally so it's fine